src/utils/fake_filesystem.cpp

```diff
@@ -65,6 +66,10 @@ timestamp_t CacheHttpfsFakeFileSystem::GetLastModifiedTime(FileHandle &handle) {
 	auto &local_filesystem_handle = handle.Cast<CacheHttpfsFakeFsHandle>().internal_file_handle;
 	return local_filesystem->GetLastModifiedTime(*local_filesystem_handle);
 }
+string CacheHttpfsFakeFileSystem::GetVersionTag(FileHandle &handle) {
```

Comment:
Fake filesystem inherits local filesystem, so if version tag fetching function unimplemented, local filesystem one will be called -- but notice, it only works for `UnixFileHandle`, instead of `FakeFileHandle`.